components/body/src/Body.js

```diff
@@ -11,7 +11,7 @@ import pkg from '../package.json';
 /**
  * Body: Body component in charge of body text
  */
-export const Body = ({ children, ...props }) => {
+export const Body = ({ tag: Tag, children, ...props }) => {
```

Comment:
Can you make sure that we include these things in the examples right when we build them?
So in the `00-typography.js` file I'd include two examples: one where you change the `div` to something like `section` and another that takes a component and passes that into the `tag` prop.

Reply:
Sure 👍

Reply:
With the latter, we talking...

```
const BodyTag = props => <span {...props} />;

...

<Body tag={BodyTag}>
   <p>Body with a component tag</p>
</Body>
```